src/components/Match/MatchStory.jsx

```diff
@@ -156,6 +156,31 @@ const formatList = (items, noneValue = []) => {
   }
 };
 
+// evaluate the sentiment behind the message - rage, question, statement etc
+const evaluateSentiment = (event, lastMessage) => {
+  const { message, player, time } = event;
+  let stringAppendature = 'normal';
+
+  if (message.split(' ').length > 10) {
+    stringAppendature = 'long';
+  } else if (message.toUpperCase() === message && /\w/.test(message)) {
+    stringAppendature = 'shouted';
+  } else if (/(\?|!|@|~|#|\$){2,}/.test(message)) {
+    stringAppendature = 'excited';
+  }
+
+  let type = message.indexOf('?') !== -1 ? 'question' : 'statement';
+  if (lastMessage && lastMessage.time + 60 > time) {
+    if (lastMessage.player.player_slot === player.player_slot) {
+      type += '_continued';
```

Comment:
consider making an array and using .join() on it to avoid having to remember where you need underscores

Reply:
Easy done.